src/app/containers/BuyBTCPage/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * BuyBTCPage
+ *
+ */
+
+import React from 'react';
+import { Helmet } from 'react-helmet-async';
+import { useTranslation } from 'react-i18next';
+import { translations } from '../../../locales/i18n';
+import { useAccount, useIsConnected } from '../../hooks/useAccount';
+import { Header } from '../../components/Header';
+import { Footer } from '../../components/Footer';
+import { openTransak } from './components/transak';
```

Comment:
Having this in `components` folder is a little bit deceitful, file content is more like provider, service or util in my eyes

Reply:
ok, maybe it's a component but then it's content is wrong - will leave comment there.